controllers/topic.controller.js

```diff
@@ -55,5 +55,23 @@ const postArticleComment = (req, res, next) => {
       }
     });
 };
+const patchUpdateVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
 
-module.exports = { getTopics, getApi, getArticleId, getArticles, getArticleIdComments, postArticleComment };
+  if (typeof inc_votes !== 'number') {
+    return res.status(400).send({ msg: 'Invalid input type' });
+  }
```

Comment:
Should deal with the errors by using error handlers. Put this logic in model would be better, psql will handle it for you